apps/web/src/pages/TalentNominations/NominationGroup/components/CurrentPositionExperiences.tsx

```diff
@@ -0,0 +1,188 @@
+import { FragmentType, getFragment, graphql } from "@gc-digital-talent/graphql";
+import { unpackMaybes } from "@gc-digital-talent/helpers";
+import { currentDate } from "@gc-digital-talent/date-helpers";
+import { MAX_DATE } from "@gc-digital-talent/date-helpers/const";
+
+import ExperienceCard from "~/components/ExperienceCard/ExperienceCard";
+
+const CurrentPositionWorkExperience_Fragment = graphql(/** GraphQL */ `
+  fragment CurrentPositionWorkExperience on WorkExperience {
+    id
+    role
+    organization
+    division
+    startDate
+    endDate
+    details
+    employmentCategory {
+      value
+      label {
+        en
+        fr
+      }
+    }
+    extSizeOfOrganization {
+      value
+      label {
+        en
+        fr
+      }
+    }
+    extRoleSeniority {
+      value
+      label {
+        en
+        fr
+      }
+    }
+    govEmploymentType {
+      value
+      label {
+        en
+        fr
+      }
+    }
+    govPositionType {
+      value
+      label {
+        en
+        fr
+      }
+    }
+    govContractorRoleSeniority {
+      value
+      label {
+        en
+        fr
+      }
+    }
+    govContractorType {
+      value
+      label {
+        en
+        fr
+      }
+    }
+    contractorFirmAgencyName
+    cafEmploymentType {
+      value
+      label {
+        en
+        fr
+      }
+    }
+    cafForce {
+      value
+      label {
+        en
+        fr
+      }
+    }
+    cafRank {
+      value
+      label {
+        en
+        fr
+      }
+    }
+    classification {
+      id
+      name {
+        en
+        fr
+      }
+      group
+      level
+      maxSalary
+      minSalary
+    }
+    department {
+      id
+      name {
+        en
+        fr
+      }
+      departmentNumber
+    }
+    workStreams {
+      id
+      key
+      name {
+        localized
+      }
+      community {
+        id
+        key
+        name {
+          localized
+        }
+      }
+    }
```

Comment:
These fields are missing in here which I think are required in this context 
    ```
supervisoryPosition
    supervisedEmployees
    supervisedEmployeesNumber
    budgetManagement
    annualBudgetAllocation
    seniorManagementStatus
    cSuiteRoleTitle {
      value
      label {
        localized
      }
    }
    otherCSuiteRoleTitle
```

Reply:
Oh shoot, I missed those when copy pasting
https://github.com/GCTC-NTGC/gc-digital-talent/pull/13338/commits/3117c1c63ada8884146889bfe0748421c191d4ec